### DISCUSSION TITLE
Ratings In "Now Playing" Bar

### DISCUSSION THREAD

**Discussion by Menghini** [navidrome/navidrome#2488] [Ideas]
One of the features I value in Navidrome is the ratings of songs. However, when I am playing a song, I cannot see or add it's ratings on the "Now Playing" bar on the bottom of the screen. The only way to rate a song is to find it in it's album and rate it. This is problematic if the album is very large.

**Comment by drtebi** [comment]
This is a great idea, I'd also love this feature.

If there are space concerns (on smaller monitors it might get cramped), this could be an optional feature that a user can turn on in his personal preferences.

**Reply by tirtawn** [reply]
I'd also love this feature since I can create a playlist with rating > 4. When you click the "Now Playing" bar, it will show the albums and some albums is quite large and I need to hunt the actual song (scrolling.....) so I could give the rating.

**Comment by zerovox** [comment]
I implemented this over here: https://github.com/navidrome/navidrome/pull/2510

**Reply by Menghini** [reply]
What a wonderful feature! I personally use both when I use Navidrome. When I go through a big album I typically click "like" on any song that doesn't aggravate me as I am typically programming or something on the side. But I forget to hit "like" on songs and of course, I'm not gonna "like" any song I don't actually like. So there will be times where I got by 10+ songs and don't know where the last song I disliked was. Could you possibly have both of them? I know your PR is like 4 months old now.. haha

**Comment by celynw** [comment]
I have (also) implemented this over here: #2879
This one is slightly different to #2510, which I only saw when writing the PR.
I agree with you that you should be able to show both the 'like' and the rating. Personally I have some 5-star songs which are/aren't favourites. They can both at least be globally enabled and disabled by config options.

Everything around this seems to be quiet and forgotten, hope it could be simple to just get merged!
